ui/webui/src/app.jsx

```diff
@@ -15,33 +15,88 @@
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
 import cockpit from "cockpit";
+
 import React, { useEffect, useState } from "react";
 
 import {
-    Page,
+    Page, Wizard,
 } from "@patternfly/react-core";
 
 import { InstallationLanguage } from "./InstallationLanguage.jsx";
-import { Summary } from "./Summary.jsx";
+import { InstallationDestination } from "./storage/InstallationDestination.jsx";
+import { ReviewConfiguration } from "./installation/ReviewConfiguration.jsx";
+import { InstallationProgress } from "./installation/InstallationProgress.jsx";
+import { InstallationComplete } from "./installation/InstallationComplete.jsx";
+
 import { AddressContext } from "./Common.jsx";
 
 import { usePageLocation } from "hooks";
 
+const _ = cockpit.gettext;
+
 export const Application = () => {
     const [address, setAddress] = useState();
     const { path } = usePageLocation();
 
     useEffect(() => cockpit.file("/run/anaconda/bus.address").watch(setAddress), []);
 
-    return (
-        <Page>
+    const wrapWithContext = children => {
+        return (
             <AddressContext.Provider value={address}>
-                {!path.length > 0 && <InstallationLanguage />}
+                {children}
             </AddressContext.Provider>
-            {path.length > 0 &&
-            <AddressContext.Provider value={address}>
-                <Summary />
-            </AddressContext.Provider>}
+        );
+    };
+
+    const steps = [
+        {
+            id: 0,
+            name: _("Installation language"),
```

Comment:
I suppose it's fine to hardcode these now... But in the future the strings and even steps should be somehow generated, same as the grid in our current hub.

Reply:
They should be probably imported from the pages directly as variables.